docs/development/astropy-package-template.rst

```diff
@@ -34,9 +34,10 @@ by ``CHANGES.md`` in the instructions.
    and update the release date, which should currently be set to
    ``unreleased``, to the current date in ``yyyy-mm-dd`` format.
 
-#. Update the version number in ``setup.cfg`` to the version you're about to
-   release, without the ``.dev`` suffix (e.g. ``0.1``). If your package uses
-   setuptools_scm to manage version numbers, you can skip this step.
+#. Update the version number to the version you're about to
```

Comment:
Ops... I think this PR fell through the cracks. Reading this again, I think this whole entry can be deleted. Using `git tag` is the way when you also use `setuptools_scm`, and this entry says you can skip this step if you use `setuptools_scm`.